docs/connector-development/connector-builder-ui/error-handling.md

```diff
@@ -44,6 +44,8 @@ Although a lot of API documentation does not call out using an exponential backo
 
 ### Wait time from header
 
+<iframe width="640" height="545" src="https://www.loom.com/embed/84b65299b5cd4f83a8e3b6abdfa0ebd2" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

Comment:
nit: I would place this under the explanation that is currently below it, since it feels like we should explain what `Wait time from header` means before showing the video for how to configure it. Not strong opinion on if it should be above or below the Example header